docs/topics/vs-code-extension-run-configuration.adoc

```diff
@@ -14,7 +14,11 @@ You can configure multiple run configurations to run against each project you im
 . Click the *+* (plus sign) next to *Migration Toolkit for Applications* to add a run configuration.
 . Complete the following configuration fields:
 
-** *Input*: Click *Add* and do one of the following:
+** *Name*: Enter a meaningful name for the analysis configuration or accept the default.
```

Comment:
@RichardHoch 
Does the default name appear in the Name field?

Reply:
@sbeskin-redhat Yes, it does

Reply:
@RichardHoch 
No, it's fine as is. I just wanted to make sure that the default name is already there.